src/components/Profile.js

```diff
@@ -70,19 +64,19 @@ const mockAllCollections = [
   },
 ];
 
-function Profile() {
+function Profile(props) {
   // myPin/ myCollection/ mySchedule(artist only)
   const MY_PIN = "myPin";
   const MY_COLLECTION = "myCollection";
   const MY_SCHEDULE = "mySchedule";
 
   const [showSection, setShowSection] = useState();
   const [pins, setPins] = useState();
+  const [userData, setUserData] = useState();
   const [collections, setCollections] = useState();
```

Comment:
initializing with '', [] or {}

Reply:
The state is used to determine if some jsx should be returned, so instead of initializing it with '[]' or '{}', I use null instead.